build_ovtf.py

```diff
@@ -322,17 +322,17 @@ def main():
                 if tags.interpreter == "cp36":
                     command_executor([
                         "pip", "install",
-                        "https://github.com/openvinotoolkit/openvino_tensorflow/releases/download/v0.5.0/tensorflow_abi1-2.4.1-cp36-cp36m-manylinux2010_x86_64.whl"
+                        "https://github.com/openvinotoolkit/openvino_tensorflow/releases/download/v0.5.0/tensorflow_abi1-2.5.0-cp36-cp36m-manylinux2010_x86_64.whl"
```

Comment:
TF-.2.5.0 ABI 1 wheel packages are not uploaded yet to git. So it will not work.